docs/docs/topics/device-identity.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Device Identity
+description: >-
+  Learn about how WebAuthn is used to authenticate and authorize access using Device ID and state.
+---
+
+## Device Identity
+
+One of the core components of a zero-trust security model is device authentication. Think of it as a component of multi-factor authentication; In addition to "what you know" (password) and "who you are" (biometrics, face recognition, etc), device authentication covers "what you have" by confirming that the device you are using to access a system is trusted.
```

Comment:
```
Think of it as a component of multi-factor authentication; In addition to "what you know" (password) and "who you are" (biometrics, face recognition, etc), device authentication covers "what you have" by confirming that the device you are using to access a system is trusted.
```

I think this is a useful explanation of the role a TEE can play in MFA; but I think we want to cover how this plays into the goals of a zero-trust model. User and device identity play a key role in being able to move beyond a trusted perimeter. ZT is about trusted device and user, not about trusted networks.

Reply:
```suggestion
One of the core components of a zero-trust security model is device authentication, whereby a device's identity and state can be factored into access control decisions. Think of it as similar to multi-factor authentication (**MFA**); where MFA covers "what you know" (password) and "who you are" (biometrics, face recognition, etc), device authentication asks "is this device safe?" by confirming that the device you are using to access a system is trusted.
```

Incomplete on its own, but a lead-in to the additional explanation you requested below.